evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -269,6 +272,33 @@ contract Operator is
     return s_withdrawableTokens.sub(ONE_FOR_CONSISTENT_GAS_COST);
   }
 
+  /**
+   * @notice Recover funds that were mistakenly sent to this contract
+   * @param recipient The address to send the LINK token to
+   * @param amount The amount to send (specified in wei)
+   */
+  function recover(address recipient, uint256 amount) 
+    external 
+    override(OracleInterface, WithdrawalInterface)
+    onlyOwner()
+    hasRecoverableFunds(amount)
+  {
+    linkToken.transfer(recipient, amount);
+  }
+
+  /**
+   * @notice Returns the amount of LINK that is recoverable (mistakenly sent to this contract)
+   * @return uint256 recoverable token amount
+   */
+  function recoverable()
```

Comment:
Do you have a scenario in mind to keep `withdrawable` and `recoverable` separate? I vote to consolidate them. I think they're close enough, and just presenting `balance - s_nonWithdrawableTokens` is all I can see people caring about. Plus, we save contract space.

Reply:
Nice, yeah that would also negate the extra gas costs associated with maintaining both.